### PR TITLE
Add pocketbase@0.22.10

### LINKED ISSUE #5748: [Request]: pocketbase
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

pocketbase

### Description

Open Source realtime backend in 1 file

### Homepage

pocketbase.io

### Download Link(s)

https://github.com/pocketbase/pocketbase/releases/download/v0.22.9/pocketbase_0.22.9_windows_amd64.zip

### Some Indication of Popularity/Repute

Stars: 33.1k
Forks: 1.5k

Open-source alternative to firebase

### PR DESCRIPTION
**Pull request by simonsan**
Closes #5748

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/pocketbase.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "bin": "pocketbase.exe",
+    "checkver": {
+        "github": "https://github.com/pocketbase/pocketbase"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/pocketbase/pocketbase/releases/download/v$version/pocketbase_$version_windows_amd64.zip"
+            }
+        }
```
Comment (aliesbelik) [inline bucket/pocketbase.json:21]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }

```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
1. Add `autoupdate.hash` section.
2. Please add `arm64` version as well.

**Comment by aliesbelik** [comment]
As from [documentation](https://pocketbase.io/docs):

> The prebuilt PocketBase executable will automatically create and manage 2 new directories alongside the executable:
    - pb_data - stores your application data, uploaded files, etc. (usually should be added in .gitignore).
    - pb_migrations - contains JS migration files with your collection changes (can be safely committed in your repository).

Wonder if at least `pb_data` is worth to be [persistent](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data) - to store between updates.

**Comment by simonsan** [comment]
> As from [documentation](https://pocketbase.io/docs):
> 
> > The prebuilt PocketBase executable will automatically create and manage 2 new directories alongside the executable:
> > - pb_data - stores your application data, uploaded files, etc. (usually should be added in .gitignore).
> > - pb_migrations - contains JS migration files with your collection changes (can be safely committed in your repository).
> 
> Wonder if at least `pb_data` is worth to be [persistent](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data) - to store between updates.

Yeah, probably both are worth to be persisted?

**Comment by simonsan** [comment]
@aliesbelik Want to take another look, if this is enough?

**Comment by aliesbelik** [comment]
Looks good to me.
But since i don't have write access in Main, please wait a bit more for merging.

**Comment by simonsan** [comment]
> Looks good to me. But since i don't have write access in Main, please wait a bit more for merging.

Yes, I'm aware. Was really just to make sure, some future person merging finds a good to go PR. :) Thanks for looking over it. 👍🏽

**Review by tech189 — APPROVED** [review]

**Pull request merged**